### PR TITLE
Adds class ovs_events plugin

### PR DESCRIPTION
**Pull request by paramite**
This patch is adding support for ovs_events plugin configuration and installation.

### CODE REVIEW

**Comment by paramite** [comment]
Can this be merged now please, when all checks are passing?

**Comment by paramite** [comment]
@alexjfisher This one is similar, can you please check it too? Thanks in advance.

File: templates/plugin/ovs_events.conf.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<%- if @address %>
+  Address "<%= @address %>"
+<%- end -%>
+<%- if @socket %>
+  Socket "<%= @socket %>"
+<%- end -%>
+<%- if @interfaces %>
+  Interfaces "<%= @interfaces.join('" "') %>"
+<%- end -%>
+<%- if defined?(@send_notification) %>
```
Comment (alexjfisher) [inline templates/plugin/ovs_events.conf.erb]: I'd have gone with `unless @send_notification.null?`, but I guess `if defined?` also works.

File: manifests/plugin/ovs_events.pp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#
+# [*socket*]
+#  The UNIX domain socket path of OVS DB server JSON-RPC interface used
+#  by the plugin
+#
+class collectd::plugin::ovs_events (
+  Optional[String] $address            = undef,
+  Optional[Boolean] $dispatch          = undef,
+  String $ensure                       = 'present',
+  Optional[Array] $interfaces          = undef,
```
Comment (alexjfisher) [inline manifests/plugin/ovs_events.pp]: Maybe better to use `Optional[Array[1]]`? or default to an empty array and adjust the template.

File: manifests/plugin/ovs_events.pp [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+#
+# [*port*]
+#  TCP-port to connect to. Either a service name or a port number may be given.
+#
+# [*socket*]
+#  The UNIX domain socket path of OVS DB server JSON-RPC interface used
+#  by the plugin
+#
+class collectd::plugin::ovs_events (
+  Optional[String] $address            = undef,
```
Comment (alexjfisher) [inline manifests/plugin/ovs_events.pp]: `Stdlib::Host` is available in recent stdlib.

**Comment by alexjfisher** [comment]
@paramite LGTM.  I've left a couple of random comments, but these are very minor and subjective.

**Comment by paramite** [comment]
@alexjfisher So as test suite discovered, "@send_notification.null?" does not work. I've added the other suggested changes.

File: manifests/plugin/ovs_events.pp [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+#
+# [*port*]
+#  TCP-port to connect to. Either a service name or a port number may be given.
+#
+# [*socket*]
+#  The UNIX domain socket path of OVS DB server JSON-RPC interface used
+#  by the plugin
+#
+class collectd::plugin::ovs_events (
+  Optional[Stdlib::Host] $address      = undef,
```
Comment (bastelfreak) [inline manifests/plugin/ovs_events.pp]: datatypes \o/

**Comment by bastelfreak** [comment]
Thanks for the PR @paramite !

**Pull request merged**